10_integrations/cloud_bucket_mount_loras.py

```diff
@@ -78,6 +85,16 @@
 )
 
 
+# For the base model, we'll use a modal.Volume to store the Hugging Face cache.
+cache_volume = modal.Volume.from_name("hf-hub-cache", create_if_missing=True)
```

Comment:
Not sure if I'm somehow muddying the message of this example by using a `Volume` when the rest of it is about `CloudBucketMount`...

Reply:
I think that's fine!